specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-04-01-preview/machineLearningServices.json

```diff
@@ -1935,6 +1935,267 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/outboundRules": {
```

Comment:
Based on the response model schema, operation 'ManagedNetworkSettingsRule_List' might be pageable. Consider adding the x-ms-pageable extension.

Reply:
done